src/fairchem/core/models/escn/escn_exportable.py

```diff
@@ -79,6 +79,7 @@ def __init__(
         resolution: int | None = None,
         compile: bool = False,
         export: bool = False,
+        rescale_grid: bool = False,
```

Comment:
unless we plan to test this, lets remove this as an option, i still prefer to remove the rescale code and keep it simpler